src/main/java/uk/gov/hmcts/reform/judicialapi/domain/RegionType.java

```diff
@@ -0,0 +1,36 @@
+package uk.gov.hmcts.reform.judicialapi.domain;
+
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.Id;
+import javax.persistence.OneToMany;
+import javax.validation.constraints.Size;
+import java.util.List;
+
+@Entity(name = "region_type")
+@Getter
+@Setter
+@NoArgsConstructor
+public class RegionType {
+
+    @Id
+    @Column(name = "region_Id")
+    @Size(max = 64)
```

Comment:
mostly we are getting with JRD APi and no insert @SIze is only used to throw exception while insert
 size > char limit. It is trivial but I feel no need in all entities